components/Attribute.vue

```diff
@@ -16,6 +19,13 @@ export default {
     },
     value() {
       return this.decode(this.attribute.value)
+    },
+    to() {
+      const addressRegexp = new RegExp(`^${BECH32_PREFIX}[a-z0-9]*$`)
```

Comment:
The regex should be improved. It currently matches addresses like: `mesgtestvaloper1mzmyqx0lr6nxj0lu4czqkgs3daa94wpeuayrcn`.
```suggestion
      const addressRegexp = new RegExp(`^${BECH32_PREFIX}[a-z0-9]{39}$`)
```

Reply:
Getting balance of `mesgtestvaloperXXX` returns an error 500:
http://localhost:3000/address/mesgtestvaloper1mzmyqx0lr6nxj0lu4czqkgs3daa94wpeuayrcn

Reply:
done